### PR TITLE
[AMDGPU] Add image_atomic_fmin/fmax as aliases for GFX12

### PR DESCRIPTION
**Pull request by jayfoad**
This just follows SP3.

### CODE REVIEW

File: llvm/lib/Target/AMDGPU/MIMGInstructions.td [hunk 1]
```diff
@@ -1675,6 +1675,10 @@ defm IMAGE_ATOMIC_PK_ADD_BF16   : MIMG_Atomic <mimgopc<0x87, MIMG.NOP, MIMG.NOP,
 defm IMAGE_ATOMIC_ADD_FLT       : MIMG_Atomic <mimgopc<0x83, MIMG.NOP, MIMG.NOP, MIMG.NOP>, "image_atomic_add_flt", 0, 1>;
 defm IMAGE_ATOMIC_MIN_FLT       : MIMG_Atomic <mimgopc<0x84, MIMG.NOP, MIMG.NOP, MIMG.NOP>, "image_atomic_min_num_flt", 0, 1, "image_atomic_min_flt">;
 defm IMAGE_ATOMIC_MAX_FLT       : MIMG_Atomic <mimgopc<0x85, MIMG.NOP, MIMG.NOP, MIMG.NOP>, "image_atomic_max_num_flt", 0, 1, "image_atomic_max_flt">;
+let AssemblerPredicate = isGFX12Plus in {
+  def : AMDGPUMnemonicAlias<"image_atomic_fmin", "image_atomic_min_flt">;
```
Comment (arsenm) [inline llvm/lib/Target/AMDGPU/MIMGInstructions.td:1679]: Do these actually have the same behavior? I've been meaning to check, I think these switched from 2008 minnum to the 2019 behavior

Comment (rampitec) [reply llvm/lib/Target/AMDGPU/MIMGInstructions.td:1679]: It's the same opcode 0x84.

**Review by rampitec — APPROVED** [review]
LGTM

File: llvm/lib/Target/AMDGPU/MIMGInstructions.td (hunk 1 above)
Comment (arsenm) [reply llvm/lib/Target/AMDGPU/MIMGInstructions.td:1679]: I don't doubt it's the same opcode, I'm questioning whether this is another surprise opcode behavior change

Comment (jayfoad) [reply llvm/lib/Target/AMDGPU/MIMGInstructions.td:1679]: Yes I think the behaviour changed (but I can't find a good spec for the pre-GFX12 behaviour).

**Pull request merged**